tests/permissions/test_permissions.py

```diff
@@ -155,3 +152,14 @@ def test_measurement_permissions_public(self):
     def test_measurement_permissions_internal(self):
         """Test that only active users can view measurements for internal stations."""
         self.assert_perms("view_measurements", self.station_internal, True, True, False)
+
+
+class StationPermissionsTestNewUser(StationPermissionsTest):
```

Comment:
It seems there are actual tests missing here, right?

Reply:
No, it inherits the tests from the parent class

Reply:
Ah, ok. Very true. I've missed the inheritance.